### PR TITLE
sending data from courses to summary with mail value

### PR DESCRIPTION
**Pull request by marcinbutora**

### CODE REVIEW

File: training-service/src/app/content/course/component/course-list/course-list.component.ts [hunk 1]
```diff
@@ -12,9 +13,12 @@ import { CourseService } from '../../service/course.service';
 })
 export class CourseListComponent implements OnInit {
 
-  course: Course[] = [];
   @Input() name: string = '';
 
+  goToSummary: string = '/summary';
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.ts]: goToSummary is a good name for a function not a variable,
summaryUrl should be a good name.

File: training-service/src/app/content/course/component/course-list/course-list.component.ts [hunk 2]
```diff
@@ -34,4 +38,21 @@ export class CourseListComponent implements OnInit {
 
   convertDurationToTime = (item: number) => item * 1000;
 
+
+  courseClick = (event: any, name: string) => {
+    if (event.target.checked) {
+      this.selectedCourses.push(name);
+      console.log(name);
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.ts]: Please remove console logs

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/content/course/component/course-list/course-list.component.ts (hunk 1 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.ts]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.ts (hunk 2 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.ts]: done

**Review by holistic-fox — APPROVED** [review]

**Pull request merged**